C_CPP/Dynamic Memory allocation/Audio_for_dynamic_memory_allocation.md

```diff
@@ -0,0 +1,19 @@
+## DYNAMIC MEMORY ALLOCATION:-
+*The process of allocating memory to the variables during execution of the program or at run time is known as dynamic memory allocation.Till now whenever we needed an array
+we had decleared a static array of fixed size,say
+int arr[100];
+When this statement is executed,consecutive space for 100 integers is allocated .It is not uncommon that we may be using only 10% or 20% of the allocated space thereby wasting
+rest of the space .To overcome this problem and to utilize the memory efficiently,C language provides a mechanism of dynamically allocating memory so that only the amount
+of memory that is actually required is reserved.We reserve space only at the run time for the variables that are actually required.Dynamic memory allocation gives best
+performance in situations in which we do not know memory requirements in advance.
```

Comment:
This should be in one line and no need for * at starting and end.

Reply:
I am not able to edit it.When I go to three dot to edit it then edit option is disabled.How can I edit it now?

Reply:
Can you please send me changes so that I can directly commit it by just clicking on commit option.